android/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/StoreProductMapper.kt

```diff
@@ -151,7 +151,7 @@ private fun Period.mapPeriod(): Map<String, Any?>? {
             "unit" to "DAY",
             "value" to 0
         )
-    }
+    } + mapOf("iso8601" to this.iso8601)
```

Comment:
Should we add it to the other mapping (for store product) as well? It shouldn't really break anything and that way we have the extra data, if we need it.

If we do, we might want to extract the key to a constant.

Reply:
The iSO 8601 version is already there in the store product version 😇